5.5/Dockerfile

```diff
@@ -32,20 +29,23 @@ RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     yum -y --setopt=tsflags=nodocs install https://www.softwarecollections.org/en/scls/rhscl/mysql55/epel-7-x86_64/download/rhscl-mysql55-epel-7-x86_64.noarch.rpm && \
     yum -y --setopt=tsflags=nodocs install gettext hostname bind-utils mysql55 && \
     yum clean all && \
-    mkdir -p /var/lib/mysql/data && chown mysql.mysql /var/lib/mysql/data && \
```

Comment:
It will be accessible even if not mounted, it is created in the `/usr/libexec/cont-setup`, that is run during the docker build. And `chmod o+w -R /var/lib/mysql` in the same file should handle the access for random uid. Maybe I don't see which use case you see to be an issue?